src/stactools/viirs/fragments/VNP09A1/collection.json

```diff
@@ -0,0 +1,65 @@
+{
+    "id": "viirs-09a1-001",
+    "title": "VIIRS M-Band (1000m) Surface Reflectance 8-Day",
+    "description": "The 8-day VIIRS surface reflectance Version 1 composite product provides an estimate of land surface reflectance from the VIIRS sensor for nine moderate resolution bands (M1 - M5, M7, M8, M10, M11) at nominal 1 kilometer resolution (~926 meter). The 1 kilometer dataset is derived through resampling the native 750 meter VIIRS resolution in the Level 2 input product. Each pixel represents the best possible Level 2G observation during an 8-day period.",
+    "license": "proprietary",
+    "links": [
+        {
+            "rel": "license",
+            "href": "https://lpdaac.usgs.gov/data/data-citation-and-policies/",
+            "type": "text/html",
+            "title": "LP DAAC - Data Citation and Policies"
+        },
+        {
+            "rel": "help",
+            "href": "https://lpdaac.usgs.gov/documents/1333/VNP09_User_Guide_V1.7.pdf",
+            "type": "application/pdf",
+            "title": "Suomi-NPP VIIRS Surface Reflectance User's Guide"
+        },
+        {
+            "rel": "cite-as",
+            "href": "https://doi.org/10.5067/VIIRS/VNP09A1.001",
+            "title": "LP DAAC - VNP09A1 v001"
+        }
```

Comment:
Should this have an `html` media type since it directs there? I'm not sure myself...

Reply:
Yeah, I think so. We are using `type` for all other links. I'm going to include this change in a future PR that's in the hopper that has all the collection fragments completed.